.github/workflows/frontend_tests.yml

```diff
@@ -37,7 +37,7 @@ jobs:
       - name: Suppress ENOSPC error from chokidar file watcher. See https://stackoverflow.com/a/32600959.
         run: echo fs.inotify.max_user_watches=524288 | sudo tee -a /etc/sysctl.conf && sudo sysctl -p
       - name: Run frontend tests
-        run: PYTHONIOENCODING=utf-8 python -m scripts.run_frontend_tests --run_minified_tests --skip_install --check_coverage
+        run: PYTHONIOENCODING=utf-8 python -m scripts.run_frontend_tests --run_minified_tests --skip_install --check_coverage --verbose
```

Comment:
Why was `--verbose` added? Most probably, I think because you are debugging the failure of frontend tests. If, so, please remove this before merging the PR.

Reply:
Removed!